source/reference/minio-mc/mc-support-inspect.rst

```diff
@@ -0,0 +1,143 @@
+======================
+``mc support inspect``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support inspect
+
+Description
+-----------
+
+:mc-cmd:`mc support inspect` allows you to collect files, including metadata files, from all backend drives based on a specified path. 
+The command produces a zip file that includes all matching files with their respective *host+drive+path*. 
+
+MinIO Engineering may request this output as part of diagnostics in |subnet|.
+
+Use the :mc-cmd:`mc support inspect` command to obtain raw object contents for analysis and upload it to the MinIO Subscription Network, |subnet|.
+Or use the command to download metadata for objects.
+
+You can read the resulting object using the MinIO debugging tool, described below.
```

Comment:
I don't think this was described below. Link to :ref:`decode tool <minio-xl-meta-decode-tool>` ?